H_ecto_models.md

```diff
@@ -243,9 +243,10 @@ Let's take a closer look at our default changeset.
     |> validate_required([:name, :email, :bio, :number_of_pets])
   end
 ```
-Right now, we have two simple pieces to this model.
+Right now, we have transformations in our pipeline to this model.
```

Comment:
Could we change it to "two transformations in our pipeline"? Sorry, I was unclear earlier.